### PR TITLE
EPMRPP-65563: fix empty stacktrace section

### PR DESCRIPTION
**Pull request by oranje322**

### CODE REVIEW

**Review by chivekrodis — CHANGES_REQUESTED** [review]
Please recheck your implementation for test without errors

File: app/src/pages/inside/common/stackTrace/stackTrace.jsx [hunk 1]
```diff
@@ -104,6 +105,13 @@ export class StackTrace extends Component {
     this.fetchItems();
   }
 
+  componentDidUpdate(prevProps) {
+    const { items, logItem } = this.props;
+    if (this.isItemsExist() === 0 && prevProps.items !== items && logItem.status === FAILED) {
+      this.fetchItems();
+    }
```
Comment (chivekrodis) [inline app/src/pages/inside/common/stackTrace/stackTrace.jsx:112]: I believe we can attach an error log to any test item or rewrite the status of a test item. So maybe `status===failed` is not the best or universal solution. Please discuss with the lead whether we will support such a case or not.

Comment (AmsterGet) [reply app/src/pages/inside/common/stackTrace/stackTrace.jsx:112]: @oranje322 Not only failed items have error logs, it's true.

File: app/src/pages/inside/common/stackTrace/stackTrace.jsx [hunk 2]
```diff
@@ -104,6 +104,13 @@ export class StackTrace extends Component {
     this.fetchItems();
   }
 
+  componentDidUpdate(prevProps) {
+    const { items, logItem } = this.props;
+    if (this.isItemsExist() === 0 && prevProps.items !== items && prevProps.logItem !== logItem) {
```
Comment (chivekrodis) [inline app/src/pages/inside/common/stackTrace/stackTrace.jsx:109]: Does it necessary check `this.isItemsExist() === 0 && prevProps.items !== items`? Looks like you should check only `logItem` or maybe better `logItem.id`

**Review by chivekrodis — CHANGES_REQUESTED** [review]

**Review by chivekrodis — APPROVED** [review]

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**